test/myFTClient.spec.js

```diff
@@ -864,4 +872,159 @@ describe('myFT Client proxy', () => {
         .catch(done);
     });
   });
+
+    //new orgainic v3/kat methods
+    describe('Followed_by_Kat concepts', () => {
+
+      afterEach(done => {
+          nock.cleanAll();
+          done();
+      });
+
+
+      const relProps = Object.assign({}, myFT.followedProperties, {byTool: 'myFTClient.spec', isTest: true});
+      const userConcepts = require('./mocks/fixtures/userFollowedConcept');
+      const katConcepts = require('./mocks/fixtures/userFollowsConceptByKat');
```

Comment:
Could these be required using?:
```javascript
const { ids, subjects } = require('./mocks/fixtures/userFollowsConceptByKat');
```
Would remove the need to refer to `katConcepts` when we want those values.